sphinx/writers/html.py

```diff
@@ -54,11 +53,8 @@ def multiply_length(length: str, scale: int) -> str:
 
 class HTMLWriter(Writer):
 
-    # override embed-stylesheet default value to 0.
-    settings_spec = copy.deepcopy(Writer.settings_spec)
```

Comment:
We have other writers modifying the `settings_spec` attribute; texinfo, text and latex. Should they be fixed also?

Reply:
> Should they be fixed also?

Yep, I'll do that now. Is it only those three?

A

Reply:
@tk0miya I just checked the `texinfo`, `text`, and `latex` writers -- these are all fine, as Docutils will have a conversion layer for a few releases. The issue is mutation of `settings_spec`.

This is therefore all that needs to change to unblock the Docutils work!

A

Reply:
Thank you for your confirmation. Now merging.